course_discovery/apps/publisher/templates/publisher/course_edit_form.html

```diff
@@ -611,6 +611,31 @@ <h1 class="hd-1 emphasized">{% trans "Edit Course" %}</h1>
                                             {% endif %}
                                     </div>
                                 </div>
+
+                                <div class="field-title">{{ form.additional_information.label }} <span class="optional float-right">Optional for announcement</span></div>
+                                <div class="row show-comparison-container">
+                                    <div class="col col-6 help-text">
+                                        <p>{% trans "Any additional information to be provided to learners." %}</p>
+                                    </div>
+                                        <div id="id_additional_information_parent" class="comparison-container">
+                                            <a id="close-comparison" class="close " href="#!">x</a>
+                                            <div id="id_additional_information_comparison" class="show-comparison-data"></div>
+                                        </div>
+                                    <div class="col col-6 help-text">
+                                        <label class="field-label ">{{ form.additional_information.label }}</label>
+                                        <input id="id_additional_information_revision" type="hidden" value="{{ history_object.additional_information }}" class="history">
+                                        {{ form.additional_information }}
+                                        <p>{% trans "2500 character limit, including spaces." %}</p>
```

Comment:
Since we do not have a limit here should we consider mentioning this is a recommended limit?  Or just leave it as it?
```suggestion
                                        <p>{% trans "Recommended 2500 character limit, including spaces." %}</p>
```